src/color.rs

```diff
@@ -10,6 +10,38 @@ use super::{BasicParseError, ParseError, Parser, ToCss, Token};
 #[cfg(feature = "serde")]
 use serde::{Deserialize, Deserializer, Serialize, Serializer};
 
+/// Implement for colors with an alpha channel.
+pub trait WithAlpha {
```

Comment:
So, `WithAlpha` is mostly an implementation detail for serialization? It seems like we don't really want to expose `serialize_alpha` to consumers, can we either make just `alpha` / `alpha_f32` built-in functions or, if it's useful to have it as a trait (is it?) just keep those, and have a

```
fn serialize_alpha(alpha: u8, dest: &mut impl fmt::Write, legacy_syntax: bool) -> fmt::Result {
```

as a standalone function?

Reply:
Simpler with a function for now.  The plan was to replace the `AbsoluteColor::alpha()` function that is used from gecko side.